src/tests/Common/scripts/bringup_runtest.sh

```diff
@@ -1276,7 +1276,6 @@ else
 fi
 
 scriptPath=$(dirname $0)
-${scriptPath}/setup-stress-dependencies.sh --arch=$ARCH --outputDir=$coreOverlayDir
```

Comment:
Why do you remove this part in the script?

Reply:
Because setup-stress-dependencies.sh has been removed, details in commit message.

Reply:
I see. The script is removed.